plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/dialogs/MultiPageWizardDialog.java

```diff
@@ -90,7 +90,9 @@ public MultiPageWizardDialog(IWorkbenchWindow window, IWizard wizard, IStructure
     }
 
     protected Point getInitialSize() {
-        return new Point(700, 500);
+        Point minSize = new Point(700, 500);
+        Point suggestedSize = super.getInitialSize();
+        return new Point(Math.max(minSize.x, suggestedSize.x), Math.max(minSize.y, suggestedSize.y));
```

Comment:
There are two unnecessary allocations.

```suggestion
        Point size = super.getInitialSize();
        size.x = Math.min(size.x, 700);
        size.y = Math.min(size.y, 500);
        return size;
```